pre_push.py

```diff
@@ -24,11 +27,14 @@ def do_process(args, shell=False):
     except Exception as exc:
         sys.stderr.write(str(exc) + "\n")
         sys.exit(1)
-    return True
+    else:
```

Comment:
Could we not add an else here please?

Reply:
I will remove the `else` statement, but I thought it added structure to the code.

Reply:
Having the else means `check_call` succeeding will result in an implicit `return None`.